contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -1887,6 +1887,10 @@ tsql_index_nulls_order(List *indexParams)
 		if (indexElem->nulls_ordering != SORTBY_NULLS_DEFAULT)
 			continue;
 
+		/* GIN indexes don't support NULLS FIRST/LAST options */
+		if (strcmp(accessMethod, "gin") == 0)
```

Comment:
Do we need to error out in here ?

Reply:
This is in order to make this feature work with Xiaohui's PR in https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1990 - in PG, GIN indexes don't support the NULL ordering which was implemented in that PR to apply to all index creation.